ding/world_model/dreamer.py

```diff
@@ -72,21 +73,26 @@ def __init__(self, cfg, env, tb_logger):
         self._cfg.norm = nn.modules.normalization.LayerNorm  # nn.LayerNorm
         self.state_size = self._cfg.state_size
         self.action_size = self._cfg.action_size
+        self.action_type = self._cfg.action_type
         self.reward_size = self._cfg.reward_size
         self.hidden_size = self._cfg.hidden_size
         self.batch_size = self._cfg.batch_size
+        if type(self.state_size) == int or len(self.state_size) == 1:
+            self.encoder = FCEncoder(self.state_size, self._cfg.encoder_hidden_size_list, activation=torch.nn.SiLU())
```

Comment:
`state_size` here must be int. List type is not acceptable in nn.Linear() at Init of FCEncoder.